extensions/stock-api-fastapi/manifest.json

```
@@ -0,0 +1,42 @@
+{
+  "version": 1,
+  "locale": "en_US.UTF-8",
+  "metadata": {
+    "appmode": "python-fastapi",
+    "entrypoint": "main"
+  },
+  "extension": {
+    "name": "stock-api-fastapi",
+    "title": "FastAPI Stock Pricing Service",
+    "description": "APIs are a great way for software engineering teams to use your models without translating them into different languages",
+    "homepage": "https://github.com/posit-dev/connect-extensions/tree/main/extensions/stock-api-fastapi",
+    "version": "0.0.0"
+  },
+  "environment": {
+    "python": {
+      "requires": ">=3.11, <4"
```

Looking at the `pyproject.toml` files for the dependencies it looks like the requirement is `requires-python = '>=3.9'`

```suggestion
      "requires": ">=3.9, <4"
```

---

After reviewing I noticed that the `README` has "Python version 3.11 or higher" I'm unsure why

---

The README came from the original Jump Start source. Are we sure we want to go against its guidance?

---

Yeah lets stick with that. I'm sure there was a reason why `3.11` was put in the README, maybe one of the imports or usages of the standard Python library requires it?

I did a quick look and didn't see anything, but still.